Cannot build with `use-dev-tty` feature

**Describe the bug**

When enabling with `use-dev-tty` feature, this crate does not build because it is missing the `rustix/process` feature:

```
error[E0433]: failed to resolve: could not find `process` in `rustix`
   --> src/event/source/unix/tty.rs:74:40
    |
74  |                 pipe::register(rustix::process::Signal::Winch as i32, sender)?;
    |                                        ^^^^^^^ could not find `process` in `rustix`
    |
note: found an item that was configured out
   --> /home/gyscos/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rustix-0.38.34/src/lib.rs:240:9
    |
240 | pub mod process;
    |         ^^^^^^^
    = note: the item is gated behind the `process` feature
```

**To Reproduce**
Steps to reproduce the behavior (from this repository):
```
cargo check --features use-dev-tty
```

**Expected behavior**
Successful compilation.

**Workaround**
Manually adding the `process` feature of the `rustix` crate makes this crate compile again.

**OS**
 - Linux

**Terminal/Console**
 - Doesn't matter

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -37,6 +37,7 @@ bracketed-paste = [
 event-stream = ["dep:futures-core", "events"] # Enables async events
 use-dev-tty = [
     "filedescriptor",
+    "rustix/process",
 ] # Enables raw file descriptor polling / selecting instead of mio.
 events = [
     "dep:mio",

